sdk/loadtestservice/azure-developer-loadtesting/src/main/java/com/azure/developer/loadtesting/LoadTestAdministrationClient.java

```diff
@@ -0,0 +1,976 @@
+
+/** Initializes a new instance of the synchronous LoadTestingClient type. */
+public final class LoadTestAdministrationClient {
+
+    @Generated private final LoadTestAdministrationAsyncClient client;
+
+    /**
+     * Initializes an instance of LoadTestAdministrationClient class.
+     *
+     * @param client the async client.
+     */
+    @Generated
+    LoadTestAdministrationClient(LoadTestAdministrationAsyncClient client) {
+        this.client = client;
+    }
+
+    /**
+     * Associate an App Component (Azure resource) to a test or test run.
+     *
+     * <p><strong>Request Body Schema</strong>
+     *
+     * <pre>{@code
+     * {
+     *     resourceId: String (Optional)
+     *     testId: String (Optional)
+     *     testRunId: String (Optional)
+     *     name: String (Optional)
+     *     value (Required): {
+     *         String (Required): {
+     *             resourceId: String (Required)
+     *             resourceName: String (Required)
+     *             resourceType: String (Required)
+     *             displayName: String (Optional)
+     *             resourceGroup: String (Optional)
+     *             subscriptionId: String (Optional)
+     *             kind: String (Optional)
+     *         }
+     *     }
+     * }
+     * }</pre>
+     *
+     * <p><strong>Response Body Schema</strong>
+     *
+     * <pre>{@code
+     * {
+     *     resourceId: String (Optional)
+     *     testId: String (Optional)
+     *     testRunId: String (Optional)
+     *     name: String (Optional)
+     *     value (Required): {
+     *         String (Required): {
+     *             resourceId: String (Required)
+     *             resourceName: String (Required)
+     *             resourceType: String (Required)
+     *             displayName: String (Optional)
+     *             resourceGroup: String (Optional)
+     *             subscriptionId: String (Optional)
+     *             kind: String (Optional)
+     *         }
+     *     }
+     * }
+     * }</pre>
+     *
+     * @param name Unique name of the App Component, must be a valid URL character ^[a-z0-9_-]*$.
+     * @param body App Component model.
+     * @param requestOptions The options to configure the HTTP request before HTTP client sends it.
+     * @throws HttpResponseException thrown if the request is rejected by server.
+     * @throws ClientAuthenticationException thrown if the request is rejected by server on status code 401.
+     * @throws ResourceNotFoundException thrown if the request is rejected by server on status code 404.
+     * @throws ResourceModifiedException thrown if the request is rejected by server on status code 409.
+     * @return app Components model along with {@link Response}.
+     */
+    @Generated
+    @ServiceMethod(returns = ReturnType.SINGLE)
+    public Response<BinaryData> createOrUpdateAppComponentsWithResponse(
+            String name, BinaryData body, RequestOptions requestOptions) {
```

Comment:
This should have `context` param.
https://azure.github.io/azure-sdk/java_introduction.html#java-service-client-context

Every API should have a simple method without options/context and another `withResponse` overload for options with context param.
Here and below.